tests/utils/helpers.py

```diff
@@ -0,0 +1,47 @@
+import subprocess
```

Comment:
maybe things like this could go into a `conftest.py` file, which is automatically sources and available for use inside the tests.